apps/frontend/src/pages/profile/libs/components/update-avatar-form/update-avatar-form.tsx

```diff
@@ -0,0 +1,54 @@
+import defaultAvatar from "~/assets/img/default-avatar.png";
+import { Button } from "~/libs/components/components.js";
+import { ZERO_INDEX } from "~/libs/constants/constants.js";
+import { ContentType } from "~/libs/enums/enums.js";
+import { useCallback, useRef } from "~/libs/hooks/hooks.js";
+import { type UserDto } from "~/modules/users/users.js";
+
+import styles from "./styles.module.css";
+
+type Properties = {
+	onSubmit: (data: FormData) => void;
+	user: UserDto;
+};
+
+const UpdateAvatarForm: React.FC<Properties> = ({
+	onSubmit,
+	user,
+}: Properties) => {
+	const fileInputReference = useRef<HTMLInputElement>(null);
+
+	const handleFormChange: React.FormEventHandler = useCallback(
+		(event: React.ChangeEvent<HTMLInputElement>): void => {
+			event.preventDefault();
+			const formData = new FormData();
+			formData.append("", event.target.files?.item(ZERO_INDEX) as File);
```

Comment:
```suggestion
			formData.append("file", event.target.files?.item(ZERO_INDEX) as File);
```
And move that name to a separate enum